inc/gui.class.php

```diff
@@ -328,6 +340,27 @@ public static function options_page() { ?>
 									</li>
 								</ul>
 							</li>
+
+							<li>
+								<input type="checkbox" name="ab_translate_api" id="ab_translate_api" value="1" <?php checked($options['translate_api'], 1) ?> />
+								<label for="ab_translate_api">
+									<?php esc_html_e('Allow comments only in certain language', 'antispam_bee') ?>
+									<span><?php esc_html_e('Detection and approval in specified language', 'antispam_bee') ?></span>
+								</label>
+
+								<ul>
+									<li>
+										<select name="ab_translate_lang">
+											<?php foreach( array('de' => 'German', 'en' => 'English', 'fr' => 'French', 'it' => 'Italian', 'es' => 'Spanish') as $k => $v ) { ?>
```

Comment:
Kein Blocker aber `esc_html_e()` oder eventuell die Sprachbezeichnungen in der jeweiligen Sprache English, Deutsch, Français?